packages/p2p/src/declarations.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '@deriv/p2p';
+declare module '@deriv/cashier';
```

Comment:
@ameerul-deriv What is this for? 🤔

Reply:
same thing i had mentioned above, but idk if we need this hahah she had mentioned to ask you if we need this or not

Reply:
@ameerul-deriv Same here, `p2p` doesn't improve itself and also doesn't have a dependency to `cashier` so this file doesn't make sense, You can remove it 🙇🏻